domain/estimators/logistic.py

```diff
@@ -74,6 +72,7 @@ def _gen_training_data(self):
         # and given TID
         self._X = torch.zeros(self.n_samples, self.num_features)
         self._Y = torch.zeros(self.n_samples)
+        self._train_idx = torch.zeros(self.n_samples)  # Keeps track of rows with NULL init_value to ignore in training.
```

Comment:
In the documentation, we need to clarify two things:
1. rows correspond to training examples not rows in terms of table rows. Might be better just to say training examples.
2. that we train only the examples that have `self._train_idx == 1`